### ISSUE TITLE
Match Mode - forcePause not possible

### ISSUE THREAD

**Issue by beckmarc** [CHR15cs/CS2-Practice-Plugin#64]
I was just checking out your plugin and I noticed a bug in the description and implementation of the match mode pause commands. Specifically, I couldn't find a way to force pause the match. In your documentation you also state to use "forceunpause" to pause and unpause the match which is not possible. 
![image](https://github.com/CHR15cs/CS2-Practice-Plugin/assets/61294771/561e25af-a63d-4b1f-aa41-ba5493d80501)

**Comment by CHR15cs** [comment]
The command should definitely be there, I will check this out

**Comment by beckmarc01** [comment]
Yes, I can but before I want to clarify in case I understand something wrong here. Is this a mistake in the documentation or are you actually supposed to use "forceunpause" even for pausing a match ?

**Comment by CHR15cs** [comment]
Ah now I think I get it, .forceunpause cannot be used to pause the match
Its meant like admins can force the unpause using .forceunpause

**Comment by beckmarc01** [comment]
That's true but my problem is that "force pause" does not work or exist. The documentation also does not contain it as you can see in my screenshot. I just want to pause the match as an admin.
